src/config/tokens/core/_core-typography.scss

```diff
@@ -11,17 +11,17 @@ $title-font-family: $font-family-english-heading, $font-family-other-languages;
 // Font definition shorthands per semantics atom definitions
 $font-h1-bold: 700 32px/40px $title-font-family;
 $font-h1-normal: 500 32px/40px $title-font-family;
-$font-h1-light: 300 24px/30px $title-font-family;
+$font-h1-light: 300 24px/40px $title-font-family;
 $font-h2-bold: 700 24px/30px $title-font-family;
 $font-h2-normal: 500 24px/30px $title-font-family;
-$font-h2-light: 300 18px/24px $title-font-family;
-$font-h3-bold: 600 18px/30px $title-font-family;
-$font-h3-normal: 500 18px/30px $title-font-family;
+$font-h2-light: 300 18px/30px $title-font-family;
+$font-h3-bold: 600 18px/24px $title-font-family;
+$font-h3-normal: 500 18px/24px $title-font-family;
 $font-h3-light: 300 18px/24px $title-font-family;
-$font-text1-bold: 700 16px/40px $title-font-family;
-$font-text1-normal: 500 16px/40px $title-font-family;
-$font-text2-bold: 700 14px/24px $text-font-family;
-$font-text2-normal: 500 14px/24px $text-font-family;
+$font-text-medium-bold: 600 16px/22px $text-font-family;
```

Comment:
Just making sure that 16px for the medium size isn't too big.
Since it's also the default value as well, it should be the most common for all cases.